examples/aws/cassandra/example.tfvars

```
@@ -1,6 +1,6 @@
 region = "ap-northeast-1"
 
-base = "default" # bai, chiku, sho
+base = "default" # dev, bai, chiku, sho
```

The default is bai ?
then 
```suggestion
base = "default" # dev, bai (default), chiku, sho
```
is more user friendly.

---

`default` is not equal `bai` currently.

Do you want the following?
```
dev, bai (default), chiku, sho
```

---

default is chiku ? then `chiku (default)`
 I'm just saying it should be more user friendly since it doesn't say what is the default.

---

I think the default is for intergration testing. (from old tools).

It would be better to clarify the definition of default and dev